grape-active_model_serializers.gemspec

```diff
@@ -24,5 +24,5 @@ Gem::Specification.new do |gem|
   gem.add_development_dependency 'rack-test'
   gem.add_development_dependency 'rake'
   gem.add_development_dependency 'guard-rspec'
-  gem.add_development_dependency 'rubocop', '0.28.0'
+  gem.add_development_dependency 'rubocop'
```

Comment:
This will cause problems. You can move it out of gemspec and into the Gemfile, but you want a version or in the future `bundle install` brings a newer one and suddenly things fail.